app/src/main/java/com/jerboa/ui/components/home/HomePane.kt

```diff
@@ -73,14 +74,15 @@ import it.vercruysse.lemmyapi.datatypes.HidePost
 import it.vercruysse.lemmyapi.datatypes.LockPost
 import it.vercruysse.lemmyapi.datatypes.MarkPostAsRead
 import it.vercruysse.lemmyapi.datatypes.PersonView
+import it.vercruysse.lemmyapi.datatypes.PostId
 import it.vercruysse.lemmyapi.datatypes.PostView
 import it.vercruysse.lemmyapi.datatypes.SavePost
 import it.vercruysse.lemmyapi.datatypes.Tagline
 import kotlinx.coroutines.launch
 
 @OptIn(ExperimentalMaterial3Api::class, ExperimentalComposeUiApi::class)
 @Composable
-fun HomeScreen(
+fun HomePane(
```

Comment:
Now that these are *panes* and not *screens*, I don't know whether its really appropriate to pass them viewmodels. I left everything as is for now.